src/wasm-binary.h

```diff
@@ -734,7 +734,8 @@ class WasmBinaryWriter {
   MixedArena allocator;
 
   Function::DebugLocation lastDebugLocation;
-  size_t lastBytecodeOffset;
+  std::vector<std::pair<size_t, const Function::DebugLocation*>> sourceMapLocations;
+  size_t sectionStartAtSourceMapLocations;
```

Comment:
I think this is the size of `sourceMapLocations` at the section start? If so, how about calling it `sourceMapLocationsSizeAtSectionStart`? Or maybe `firstSourceMapLocationInSection`?